src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -43,6 +36,9 @@
  * running on Kafka Connect Workers.
  */
 public class SnowflakeSinkConnector extends SinkConnector {
+  // create logger without correlationId for now
+  private static LoggerHandler LOGGER = new LoggerHandler(SnowflakeSinkConnector.class.getName());
```

Comment:
This is how all the LOGGERs will be instantiated. Since they are static, they are created with no correlationId (it will be set later on)